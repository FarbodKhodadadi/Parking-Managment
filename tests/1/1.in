request_spot Honda_Fit
request_spot Peraid
request_spot Samand
request_spot Peraid
request_spot Ford_Expedition
