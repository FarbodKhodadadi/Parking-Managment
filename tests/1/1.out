101: normal 250 50
255: CCTV 330 110
245: normal 300 70
301: covered 350 100
201: CCTV 430 140
321: normal 350 80
443: covered 400 110
245: normal 300 70
301: covered 350 100
201: CCTV 430 140
321: normal 350 80
443: covered 400 110
